app/code/BodenkoVV/AskQuestion/Controller/Adminhtml/Question/InlineEdit.php

```diff
@@ -0,0 +1,139 @@
+ */
+class InlineEdit extends \Magento\Backend\App\Action
+{
+    /** Authorization level of a basic admin session    */
+    const ADMIN_RESOURCE = 'BodenkoVV_AskQuestion::save';
+
+    /** @var \Magento\Cms\Controller\Adminhtml\Page\PostDataProcessor */
+    protected $dataProcessor;
+
+    /** @var \Magento\Cms\Api\PageRepositoryInterface */
+    protected $pageRepository;
+
+    /** @var \Magento\Framework\Controller\Result\JsonFactory */
+    protected $jsonFactory;
+
+    /**
+     * @param Context $context
+     * @param JsonFactory $jsonFactory
+     */
+    public function __construct(
+        Context $context,
+        JsonFactory $jsonFactory
+    ) {
+        parent::__construct($context);
+        $this->jsonFactory = $jsonFactory;
+    }
+
+    /**
+     * @return \Magento\Framework\Controller\ResultInterface
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function execute()
+    {
+        /** @var \Magento\Framework\Controller\Result\Json $resultJson */
+        $resultJson = $this->jsonFactory->create();
+        $error = false;
+        $messages = [];
+
+        $postItems = $this->getRequest()->getParam('items', []);
+        if (!($this->getRequest()->getParam('isAjax') && count($postItems))) {
+            return $resultJson->setData([
+                'messages' => [__('Please correct the data sent.')],
+                'error' => true,
+            ]);
+        }
+
+        foreach (array_keys($postItems) as $pageId) {
+            /** @var \BodenkoVV\AskQuestion\model\Question $page */
+            $page = $this->pageRepository->getById($pageId);
+            try {
+                $pageData = $this->filterPost($postItems[$pageId]);
+                $this->validatePost($pageData, $page, $error, $messages);
+                $extendedPageData = $page->getData();
+                $this->setCmsPageData($page, $extendedPageData, $pageData);
+                $this->pageRepository->save($page);
+            } catch (\Magento\Framework\Exception\LocalizedException $e) {
+                $messages[] = $this->getErrorWithPageId($page, $e->getMessage());
+                $error = true;
+            } catch (\RuntimeException $e) {
+                $messages[] = $this->getErrorWithPageId($page, $e->getMessage());
+                $error = true;
+            } catch (\Exception $e) {
+                $messages[] = $this->getErrorWithPageId(
+                    $page,
+                    __('Something went wrong while saving the page.')
+                );
+                $error = true;
+            }
+        }
+
+        return $resultJson->setData([
+            'messages' => $messages,
+            'error' => $error
+        ]);
+    }
+
+    /**
+     * Filtering posted data.
+     *
+     * @param array $postData
+     * @return array
+     */
+    protected function filterPost($postData = [])
+    {
+        $pageData = $this->dataProcessor->filter($postData);
```

Comment:
This variable is not injected in constructor. Do you really need this filter? I would advice to remove it, please investigate, explain why it is needed and decide if it need to be fixed.